script.module.dropbox/addon.xml

```diff
@@ -1,33 +1,36 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="script.module.dropbox"
        name="Dropbox API"
-       version="8.4.1"
+       version="8.4.2"
        provider-name="micah">
   <requires>
     <import addon="xbmc.python" version="2.25.0"/>
     <import addon="script.module.requests" version="2.18.4"/>
-    <import addon="script.module.qrcode" version="5.2"/> <!-- For QR Code display for Dropbox API authentication -->    
+    <import addon="script.module.six" version="1.11.0" />
+    <import addon="script.module.qrcode" version="5.2" optional="true" /> <!-- For QR Code display for Dropbox API authentication -->
   </requires>
-  <extension point="xbmc.python.library" library="authenticate.py"> <!-- entry to start the authentification script from the settings dialog of other addons -->
-    <provides> </provides> <!-- Addon should not be shown anywhere -->
-  </extension>
+  <extension point="xbmc.python.library" library="authenticate.py" /> <!-- entry to start the authentification script from the settings dialog of other addons -->
   <extension point="xbmc.python.module" library="lib" /> <!-- entry to import this in other addons -->
   <extension point="xbmc.addon.metadata">
     <summary lang="en">Dropbox API</summary>        
     <description lang="en">Provide Dropbox Python API support</description>
     <description lang="he">התוסף מספק תמיכה ב-API Python Dropbox</description>
-    <disclaimer lang="en"></disclaimer>     
     <platform>all</platform>
     <language/>
     <license>Apache2</license>
-    <forum></forum>
-    <website></website>
-    <source>http://python-requests.org</source>
+    <website>https://www.dropbox.com</website>
+    <source>https://www.dropbox.com/developers/documentation/python</source>
     <email>micahgalizia AT gmail.com</email>
```

Comment:
upstream email is `dev-platform@dropbox.com`

Reply:
Usually I would set this to the upstream email, or do not provide the email tag. If you want to keep your own email here, it's fine for me.